### PR TITLE
Derivation rules by SATySFi

### PR DESCRIPTION
**Pull request by amutake**
- added SATySFi version of derivation-rules.md
- auto build & push to gh-pages

### CODE REVIEW

File: docs/derivation-rules.saty [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+            | \judgement{A}{m}{\tau_m}{C_m} |
+              \judgement{A}{f}{\tau_f}{C_f} |
+              \judgement{A}{a}{\tau_a}{C_a} |
+          }{
+            \judgement{A}{\mathtt{fun}\ \mfa{m}{f}{a}}{\beta}{\(\tau_m \subseteq \mathtt{atom\(\)}\) \wedge \(\tau_f \subseteq \mathtt{atom\(\)}\) \wedge \(\tau_a \subseteq \mathtt{number\(\)}\) \wedge C_m \wedge C_f \wedge C_a}
+          }
+        }
+      );
+    >
+    +small-block<+math(${\diamond: \text!{neither ${m}, ${f} is atom literal nor ${a} is non_neg_integer literal}});>
```
Comment (amutake) [inline docs/derivation-rules.saty]: To prevent overhang...😢

File: docs/derivation-rules.saty [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+            \judgement{A}{p\ \mathtt{when}\ g}{\tau}{\(\tau \subseteq \mathtt{boolean\(\)}\) \wedge C_p \wedge C_g}
+          }
+        }
+      );
+    >
+    +small-block<
+      +math(
+        ${
+          \derive?:{\paren{\mathsc{CASE}}}{
+            | \judgement{A}{p}{\tau}{C_e} |
```
Comment (yoshihiro503) [inline docs/derivation-rules.saty]: Is it a typo?
```suggestion
            | \judgement{A}{e}{\tau}{C_e} |
```

Comment (amutake) [reply docs/derivation-rules.saty]: It is a typo! Thanks!

Comment (amutake) [reply docs/derivation-rules.saty]: fixed https://github.com/dwango/fialyzer/pull/205/commits/551ebc7d6d811bc4047255f97e49b407a8d36f80

**Pull request merged**